src/components/clickable-image/ClickableImage.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+
+import { Box } from '@mui/material'
+
+import AppButton from '~/components/app-button/AppButton'
+
+import { ButtonVariantEnum } from '~/types'
+
+import { styles } from '~/components/clickable-image/ClickableImage.styles'
+
+interface ClickableImageProps {
+  clickFunction?(image: string): void
```

Comment:
onClick?: (image: string) => void
 better name it onClick instead of clickFunction

Reply:
> onClick?: (image: string) => void better name it onClick instead of clickFunction

fixed